### PR TITLE
Add postgresql and postgresql with PITR apps in integration test suite

### PR DESCRIPTION
**Pull request by PrasadG193**
## Change Overview

- Rename existing postgres app to rds-postgres
- Use kanister logging across integration tests
- Add postgres app with PITR in integration tests
- Add plain postgres app in integration tests
- Increase integration test timeout to 50m

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [x] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [x] :robot: Test

## Issues

- #XXX

## Test Plan

To run:
```
go test -v -tags=integration -timeout 50m ./pkg/testing/
```

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [x] :green_heart: E2E

### CODE REVIEW

File: pkg/app/bp.go [hunk 1]
```diff
@@ -16,33 +16,60 @@ package app
...
+	return &PITRBlueprint{
+		AppBlueprint{
+			app:  app,
+			path: fmt.Sprintf("%s/%s-blueprint.yaml", blueprintsRepo, app),
+		},
+	}
+}
+
+func (b PITRBlueprint) FormatPITR(pitr time.Time) string {
+	loc, err := time.LoadLocation("GMT")
```
Comment (PrasadG193) [inline pkg/app/bp.go]: Use time.UTC

File: pkg/app/postgresql.go [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+	values   map[string]string
+}
+
+type PostgresDB struct {
+	cli           kubernetes.Interface
+	chart         chartInfo
+	password      string
+	namespace     string
+	podName       string
+	containerName string
```
Comment (PrasadG193) [inline pkg/app/postgresql.go]: Remove this

File: pkg/testing/integration_test.go [hunk 3]
```diff
@@ -113,12 +135,12 @@ func (s *IntegrationSuite) TestRun(c *C) {
 	defer cancel()
 
 	// Execute e2e workflow
-	log.Infof("Running e2e integration test for %s", s.name)
+	log.Info().Print("Running e2e integration test.", field.M{"app": s.name})
 
 	// Check config
```
Comment (PrasadG193) [inline pkg/testing/integration_test.go]: Do profile validation at start

File: pkg/blueprint/blueprints/pitr-postgres-blueprint.yaml [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+              # Region is required when no endpoint is used (AWS S3).
+              wale_s3_region="us-east-1"
+              {{- if .Profile.Location.Region }}
+                wale_s3_region="{{ .Profile.Location.Region | quote}}"
+              {{- end }}
+              echo "${wale_s3_region}" > "${env_wal_region}"
+            {{- end }}
+
+            set +o xtrace
+            echo "{{ .Profile.Credential.KeyPair.Secret }}" > "${env_wal_secret_access_key}"
```
Comment (SupriyaKasten) [inline pkg/blueprint/blueprints/pitr-postgres-blueprint.yaml]: @PrasadG193 we made a few changes to pg blueprint, please take a look at this https://github.com/kanisterio/kanister/blob/master/examples/helm/kanister/kanister-postgresql/kanister/postgres-blueprint.yaml#L73

Comment (PrasadG193) [reply pkg/blueprint/blueprints/pitr-postgres-blueprint.yaml]: @SupriyaKasten I have addressed your comments and opened seperate PR https://github.com/kanisterio/kanister/pull/428

**Comment by PrasadG193** [comment]
Spliting this PR into 2 seperate PRs-
https://github.com/kanisterio/kanister/pull/427
https://github.com/kanisterio/kanister/pull/428
Hence closing this PR

**Pull request closed without merging**